### ISSUE TITLE
Getting error when trying to bootrun with Gradle.  Grails 6.1.2/Spring Boot 3.2.2/Spring 6.1.3/Groovy 3.0.19

### ISSUE THREAD

**Issue by dgeiselmanMOO** [grails/grails-core#13381]
### Expected Behavior

Should run the application so that I can view it through Tomcat

### Actual Behaviour

Exception in thread "restartedMain" java.lang.reflect.InvocationTargetException
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'grailsApplicationPostProcessor' defined in uc.Application: Failed to instantiate [grails.boot.config.GrailsApplicationPostProcessor]: Factory method 'grailsApplicationPostProcessor' threw exception with message: org/springframework/core/type/classreading/AnnotationMetadataReadingVisitor

### Steps To Reproduce

I'm using Grails 6.1.2, Spring 6.1.3, Spring Boot 3.2.2, Groovy 3.0.19.  Happens when after I build my build.gradle and then run the bootrun command to load my project in Tomcat.   This is happening after upgrading from Grails 5.3, Spring Boot 2+, Spring 5+, Groovy 3.0.7

### Environment Information

Windows 10, JDK17

### Version

6.1.2

**Comment by matrei** [comment]
Hi @dgeiselmanMOO, Grails 6 is still on Spring 5.3, Spring Boot 2.7.
https://docs.grails.org/6.1.2/guide/introduction.html#whatsNew

Grails 7 will upgrade to later major versions of these dependencies.

**Comment by dgeiselmanMOO** [comment]
That could be bad, I have to upgrade to Spring 3 for vulnerability fixes

**Comment by matrei** [comment]
Yep, that can be a problem. Spring Boot OSS support periods are really short.
They still have commercial support for Spring Boot 2.7 though:
https://endoflife.date/spring-boot

**Comment by dgeiselmanMOO** [comment]
Changing to those versions did resolve this issue.  Thank you

**Comment by dauer** [comment]
> Changing to those versions did resolve this issue. Thank you

Issue seems to be resolved - I will close it
